src/screens/BrowseSourceScreen/BrowseSourceScreen.tsx

```diff
@@ -119,8 +119,8 @@ const BrowseSourceScreen: React.FC<BrowseSourceScreenProps> = ({ route }) => {
         rightIcons={[{ iconName: 'earth', onPress: handleOpenWebView }]}
         theme={theme}
       />
-      {isLoading || isSearching ? (
-        <LoadingScreenV2 theme={theme} />
+      {isLoading || isSearching || false ? (
```

Comment:
```suggestion
      {isLoading || isSearching ? (
```

Reply:
Sorry, I forgot to commit my last changes which were basically your two remarks.